collector/hyperv.go

```diff
@@ -593,6 +605,69 @@ func NewHyperVCollector() (Collector, error) {
 			[]string{"vm_interface"},
 			nil,
 		),
+
+		//
+
+		VMMemoryAddedMemory: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, buildSubsystemName("vm_memory"), "added"),
+			"",
+			[]string{"vm"},
+			nil,
+		),
```

Comment:
Each of the `Counter` metrics will need a `_total` suffix added to the description.